src/utilities/messageFormatter.ts

```diff
@@ -129,7 +150,28 @@ export const getMediaTypeFromExtension = (path: string): MediaType => {
   }
 };
 
-export const getAbsoluteFilePath = (path: string) => {
+const convertExtension = (file?: Asset): string => {
+  if (!file || file.type?.startsWith('image')) {
+    return 'jpg';
+  } else {
+    return 'mp4';
+  }
+};
+
+const getAbsoluteFilePath = (path: string) => {
+  return path?.startsWith?.('file:/') ? path : `file://${path}`;
+};
+
+const getAbsoluteFilePathWithName = async (path: string, name: string) => {
+  if (path.startsWith('content://') && name) {
+    try {
+      const destPath = `${RNFS.TemporaryDirectoryPath}/${name}`;
+      await RNFS.copyFile(path, destPath);
+      path = destPath;
+    } catch (error) {
+      console.error('Error copying file:', error);
+    }
+  }
   return path?.startsWith?.('file:/') ? path : `file://${path}`;
```

Comment:
can we use `getAbsoluteFilePath` function above here?

Reply:
ok I updated